### ISSUE TITLE
Is it possible to add to the wiki advice on rupy execution settings?

### ISSUE THREAD

**Issue by fgd99** [tinspin/rupy#2]
Most particularly on how to run rupy as a service, with an automatic restart.
For the moment I have tested to do it with an init.d script like this http://stackoverflow.com/questions/11203483/run-a-java-application-as-a-service-on-linux

**Comment by tinspin** [comment]
Sure, can you paste the working file you have here https://gist.github.com and reply with the url?

**Comment by fgd99** [comment]
Yes of course, here it is https://gist.github.com/fgd99/0d1849674968f6ae408a

But do you have a better way to do this? Like a service wrapper of anything else that can automatically restart the rupy process?

**Comment by tinspin** [comment]
Thanks! I think this is the best way to do it. What is your user case? Maybe I can help you from a "out-of-the-box" perspective if you describe your setup/problem?

I'm using DNS round robin across a cluster, distributed over multiple physical locations so I'm not as concerned with automatic reboot (I can boot manually in the rare case of a local problem) and if I was having alot of power issues I would probably use a 12v lead-acid battery to avoid instance reboot altogether.

**Comment by fgd99** [comment]
I have little experience with java web servers in production and I was wondering how to run rupy like I do with node.js with forever (https://github.com/foreverjs/forever) or pm2 (https://github.com/Unitech/pm2).

Thank you for this project!

**Comment by tinspin** [comment]
Ok, interesting. I hope you will appreciate Java more using rupy.

The reason I'm not a fan of computers doing things automatically (like terminator for example ;) is that they can sometimes go into some recursive loop that is hard to understand because you can't intuitively sense (visualize) what is occurring since the machines are far away (latency).

If that involves storage or integration operations, things can quickly cascade out of control in a massively concurrent and deep hierarchy micro services architecture.

All the benefits of the mentioned tools can be had with distribution instead, which also has the benefit of scaling aswell as being redundant and providing instant backup, all at the same time.

Try my distributed JSON database if you're interested: http://root.rupy.se
